File: 136-single-number/single-number.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function singleNumber($nums) {
        sort($nums);
        $n = sizeof($nums);
        for($i=0; $i<$n; $i+=2){
            if($i === $n-1)
                return $nums[$i];
            if($nums[$i] != $nums[$i+1])
                return $nums[$i];
        }
    }
}

File: 136-single-number/single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        Arrays.sort(nums);
        int n = nums.length;
        for(int i=0; i<n; i+=2){
            if(i == n-1)
                return nums[i];
            if(nums[i] != nums[i+1])
                return nums[i];
        }
        return 0;
    }
}
